demo/image_creation.sh

```diff
@@ -1,125 +1,162 @@
 #!/bin/bash
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
+#
```

Comment:
Ideally this should go into a -h message, but I'm ok with improving it later

Reply:
Done, added command-line options and defined USAGE.